### DISCUSSION TITLE
Migrating from strimzi certificates to our own certificates

### DISCUSSION THREAD

**Discussion by raghavendra-bhat-maersk** [strimzi/strimzi-kafka-operator#7785] [Q&A] [answered]
Hi,

We want to replace strimzi certificates by my own certificates in production cluster, where we have clients running.
Could you please let us know if there is any way to migrate from strimzi certificates to our own certificates without cluster downtime?

From the documentation I can see that there should not be any cluster deployed before creating our own certificates.

<img width="987" alt="image" src="https://user-images.githubusercontent.com/71883431/207031320-a1d2474e-0c58-4a06-a282-fe2fb805b2cd.png">

Regards,
Raghav

**Comment by scholzj** [comment]
@ppatierno was working on that. Maybe he can provide some help.

**Comment by ppatierno** [comment, marked as answer]
I think that the documentation section you are looking for is the following:
https://strimzi.io/docs/operators/latest/full/configuring.html#proc-replacing-your-own-private-keys-str

One thing to take into account is that the above documentation is referring to the following use case: you have a Kafka cluster with your own custom CA cert and key and you want to renew both CA cert and key.

Your use case is different, because you are starting with Strimzi operator CA cert and key and you want to use custom CA cert and key.

Even if use case sounds different, the steps are the same. Just take into account that the doc should say "replace" instead of "renew".
We see this as a problem in the doc that we are going to address soon in order to make this "replacement" use case clearer to the users.
